src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyFactory.java

```diff
@@ -140,6 +140,11 @@ public HttpCacheProxyFactory(
         this.executor = executor;
     }
 
+    public Int2ObjectHashMap<HttpProxyCacheableRequestGroup> getRequestGroups()
+    {
+        return requestGroups;
+    }
+
```

Comment:
This is approximately the same as making the field public. :-)

Perhaps we could instead just turn this into `public HttpProxyCacheableRequestGroup  getRequestGroup(int requestHash)` ?

Reply:
:)